artemVMV/scrimbaModule6/vite.config.js

```diff
@@ -0,0 +1,6 @@
+import {defineConfig} from 'vite'
```

Comment:
Wow, nice that you have used  vite, was it a part of a course or your initiative?